.github/workflows/scripts/.pinot_test.sh

```diff
@@ -44,17 +44,9 @@ if [ $? -eq 0 ]; then
   fi
```

Comment:
line 38-44 can also be simplified as:
```suggestion
  mvn test && exit 0 || exit 1
```